content/lessons/genesis/messages.js

```diff
@@ -0,0 +1,16 @@
+import { defineMessages } from 'react-intl'
+import { formatMessages } from 'lib/messages'
+
+const messages = defineMessages({
+  blockExplorer: 'block explorer',
+  blockExplorerTooltipContent: 'We recommend https://blockstream.info/',
+  firstParagraph:
+    'Let’s find the very first block in the Bitcoin blockchain. Go to a {blockExplorerTooltip} and search for ’0’. This will take you to the block at height 0 you can see, this block only contains one transaction.',
+})
+
+export default messages
+
+export const formattedMessages = formatMessages(
```

Comment:
Every messages.js file should follow a pattern like this with a default export from the `defineMessages` results and a an export called `formattedMessages` which is the result from `formatMessages`.

I made the file a JS file cause I couldn't figure out how to make the TS gods happy. These files are pretty basic, so I don't think it's a big deal to leave them as JS.